modules/installation-azure-permissions.adoc

```diff
@@ -7,15 +7,11 @@
 [id="installation-azure-permissions_{context}"]
 = Required Azure roles
 
-{product-title} needs a service principal so it can manage Microsoft Azure resources. Before you can create a service principal, review the following information:
+An {product-title} requires an Azure identity to create and manage Azure resources. Before you create the identity, verify that your environment meets the following requirements:
```

Comment:
```suggestion
An {product-title} cluster requires an Azure identity to create and manage Azure resources. Before you create the identity, verify that your environment meets the following requirements:
```

Reply:
Hello @mjpytlak,
       My only comment/suggestion would be to match the verbiage with the paragraph above, which basically states the same thing. That being said, I am not sure if this will change dependent on OCP doc versions, so may not be relevant. Thanks. 
![image](https://github.com/openshift/openshift-docs/assets/122639474/39474116-413b-4e31-ac03-540f2098f2e7)